GitHub Action pnpm version conflict: Action config vs packageManager field

**Issue:**  
The GitHub Actions workflow is failing with a version conflict when trying to set up pnpm. The error says two different pnpm versions are specified:

1. `version 10` in the GitHub Action config (inside `.github/workflows/*.yml`)
2. `version pnpm@10.16.0` in `package.json` under the `"packageManager"` field

This mismatch causes the action `pnpm/action-setup@v4` to fail during the setup step.

**Error log snippet:**
```
Error: Multiple versions of pnpm specified:
  - version 10 in the GitHub Action config with the key "version"
  - version pnpm@10.16.0 in the package.json with the key "packageManager"
Remove one of these versions to avoid version mismatch errors like ERR_PNPM_BAD_PM_VERSION
```

**Why it matters:**  
The workflow won’t run until this is resolved—meaning no CI/CD, no automatic migrations, no deployments.

--- a/.github/workflows/database_migrations.yml
+++ b/.github/workflows/database_migrations.yml
@@ -20,8 +20,6 @@ jobs:
         uses: actions/checkout@v4
       - name: Install pnpm
         uses: pnpm/action-setup@v4
-        with:
-          version: 10
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:

